keyboard.html

```diff
@@ -142,7 +142,7 @@ <h2 id="others">What are the additional benefits?</h2>
       </ul>
 
       <h2 id="action">What needs to happen for this to work?</h2>
-      <p>Native <abbr title="Hypertext Markup Language">HTML</abbr> controls, like links, buttons, and form elements, work with the keyboard by default. Custom-made controls, <abbr title="Cascading Style Sheets">CSS</abbr> styles, and scripts that control interaction may need additional coding for keyboard compatibility. Markup the tab order so that navigation around the content and the controls is logical. Provide a way for users to jump between groups of related controls. Keyboard issues occur particularly in forms, menus, and applications with many controls.</p>
+      <p>Native <abbr title="Hypertext Markup Language">HTML</abbr> controls, like links, buttons, and form elements, work with the keyboard by default and shuld be used where possible. Custom-made controls, <abbr title="Cascading Style Sheets">CSS</abbr> styles, and scripts that control interaction may need additional coding for keyboard compatibility. Markup the tab order so that navigation around the content and the controls is logical. Provide a way for users to jump between groups of related controls. Keyboard issues occur particularly in forms, menus, and applications with many controls.</p>
```

Comment:
I think it would be easier to read this paragraph with a list structure, instead of giving a list in a long sentence.

Also, typo "shuld" should be "should"